games-puzzle/icebreaker/additional-files/icebreaker-1.9.7.rdef

```diff
@@ -0,0 +1,74 @@
+
+resource app_flags B_SINGLE_LAUNCH | B_ARGV_ONLY;
+
+resource app_version {
+       major  = 1,
+       middle = 9,
+       minor  = 7,
+
+       variety = B_APPV_FINAL,
+       internal = 0,
+
+       short_info = "IceBreaker",
+       long_info = "An addictive action-puzzle game"
```

Comment:
@Begasus, I've just noticed the indentation in the app_version block block is made of white spaces. I guess you will want to use tabs instead.